.travis.yml

```diff
@@ -6,7 +6,7 @@ jdk:
 env:
   matrix:
   - CI_TEST: ci-fast
-    CI_SCALA_VERSION: 2.11.11
+    CI_SCALA_VERSION: 2.12.3
```

Comment:
This disables ci for 2.11, can we move ci_publish to 2.12 entry instead?

Reply:
doesn't the CI run anyway (although in "slow" mode?)

Reply:
anyway, yes, your solution makes much more sense. Done.